src/app.js

```diff
@@ -1 +1,47 @@
+/* eslint-disable no-console */
 'use strict';
+
+const { terminal } = require('./terminal');
+const fs = require('fs');
+const path = require('path');
+
+const promptCopyFile = () => {
+  terminal.question(
+    'What must be copied? (Example: "cp file.txt file-copy.txt"): ',
+    (input) => {
+      const [ command, source, dest ] = input.split(' ');
+
+      if (command !== 'cp' || !source || !dest) {
```

Comment:
```suggestion
      if (!source || !dest) {
```

Fix the error message to be more specific. You can split it into two parts: one for source, and one for dest errors.